tests/integration/tier0/basic-sanity-checks/test_basic_sanity_checks.py

```diff
@@ -112,4 +115,16 @@ def test_c2r_latest_older_unsupported_version(convert2rhel):
 
     # Clean up
     os.system(f"cp /tmp/__init__.py {PATH_TO_VERSION}")
+    os.system(f"cat {PATH_TO_VERSION}")
```

Comment:
and here?

Reply:
Ok, thank you for spotting those, I'll make sure to remove those lines of clean up code!

Reply:
Oh, good catch! You're right, it's gone now